content/articles/2017-12-nov-updates-2017.Rmd

```diff
@@ -0,0 +1,57 @@
+---
+title: November happenings in the tidyverse
+slug: nov-updates-2017
+date: '2017-12-11'
+author: Mara Averick
+categories:
+  - other
+photo:
+  url: https://unsplash.com/photos/lPQIndZz8Mo
+  author: Pablo Garcia Saldaña
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+In an effort to keep the community up to date with the evolution of the tidyverse, we'll be doing monthly roundups cataloging the latest developments.
+
+## tidyverse package updates
+
+* [tidyverse 1.2.0 `r icon::fa("file")`](https://www.tidyverse.org/articles/2017/11/tidyverse_1.2.0/)  ([release notes](https://github.com/tidyverse/tidyverse/releases/))
+  + membership changes, with [stringr](http://stringr.tidyverse.org) and [forcats](http://forcats.tidyverse.org) moving to the core, and the addition of the [reprex](http://reprex.tidyverse.org/) package
+
+* [lubridate 1.7.0 `r icon::fa("file")`](https://www.tidyverse.org/articles/2017/11/lubridate-1.7.0/) ([release notes](https://github.com/tidyverse/lubridate/releases))
+
+* rlang 0.1.4 ([release notes](https://github.com/tidyverse/rlang/releases))
+
+* tidyselect 0.2.3 ([release notes](https://github.com/tidyverse/tidyselect/releases))
+
+### tidyselect
+
+tidyselect provides a common back-end for `dplyr::select()`, `tidyr::gather()`, as well as for modelling packages. It is also the source of selection helpers, such as `everything()`, and `starts_with()`. tidyselect allows you to create selecting verbs that are consistent across tidyverse packages.
+
+### tidy models
+
+[yardstick](https://topepo.github.io/yardstick/) allows you to easily create tidy performance estimates. Using a syntax similar to dplyr’s you can compute common performance metrics, such as precision, and recall (for classification), or numeric metric outcomes for regression, and have them returned in a tidy data frame. 
+
+[recipes](https://topepo.github.io/recipes/) is an extensible framework for feature selection and the creation of preprocess design matrices, which can then be applied to statistical and machine learning models. The updated version of recipes includes a tidy method for many of the step functions. The tidy method returns relevant information about the step. This could include estimated parameters or which variables were affected by the step.
```

Comment:
There were a bunch of new steps added too. Can you link to the NEW files for [`recipes`](https://topepo.github.io/recipes/news/index.html) and [`rsample`](https://topepo.github.io/rsample/news/index.html)?